tmpl/email_html.mustache

```diff
@@ -3,21 +3,20 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Portier &ndash; Finish logging in to {{ client_id }}</title>
+    <title>Portier &ndash; {{ title }} {{ client_id }}</title>
   </head>
   <body style="font: normal 1em/1.25em sans-serif">
     <div style="margin: 72px auto; max-width: 640px; text-align: center">
       <p style="margin: 24px; font:normal 1.25em sans-serif">
-        You received this email so that we may confirm your email address
-        and finish your login to: <em>{{ client_id }}</em>
+        {{ explanation }} <em>{{ client_id }}</em>
```

Comment:
The normal solution would be to give the variable to the gettext helper, with the translation placing the variable content where it belongs. Would be ugly here because of the html, but that can be ignore if necessary, but… frankly, I'm not sure whether https://docs.rs/gettext/0.2.0/gettext/ supports it, I don't see it clearly in the docs.